documentation/manual/src/doc/algebras/json-schemas.md

```diff
@@ -118,9 +118,22 @@ reduced to the following:
 ~~~
 
 The `genericJsonSchema` operation builds a JSON schema for the given
-type, mapping each case class field to a JSON object field of the same name,
-and each leaf of a sealed trait to a tagged JSON object whose type discriminator
-uses the case class fully qualified name.
+type. The rules for deriving the schema are the following:
+
+- the schema of a case class is a JSON object,
+- the schema of a sealed trait is the alternative (`oneOf`) of its leaf case
+  class schemas, discriminated by the case class names,
```

Comment:
Maybe you could mention that overriding 'classTagToSchemaName' is now supported, if relevant?

Reply:
Indeed, I should document that. I was thinking of doing it in another PR, where I will also introduce a way to customize the name of a generically derived schema.